pages/identities/[tokenId].tsx

```diff
@@ -57,39 +65,110 @@ const TokenIdPage: NextPage = () => {
     }
   }, [tokenId]);
 
+  // Check if user has a pfp set
+  useEffect(() => {
+    if (!tokenId) return;
+
+    const getVerifierData = async () => {
```

Comment:
Could we move that also ?

Reply:
This one might be a bit more complicated to move to another file, but right now I'm updating `starknetid.js` to have a function that will return the verifier data for the profile picture, so it will reduce this function and make it way more readable.